src/CMakeLists.txt

```diff
@@ -44,6 +43,7 @@ set (ALBANY_INCLUDE_DIRS
      ${Albany_SOURCE_DIR}/src/disc/stk
      ${Albany_SOURCE_DIR}/src/disc
      ${Albany_SOURCE_DIR}/src/utility
+     ${Albany_SOURCE_DIR}/src/corePDEs/evaluators
```

Comment:
A small nitpick but I would probably call this directory `CorePDEs` and the other one `DemoPDEs` to stick with the very loose convention of using CamelCase (e.g. `LandIce`). Probably better than using snake_case which is used for all other directories so we avoid having to change LandIce.